src/reports.rs

```diff
@@ -124,24 +124,29 @@ impl DependencyReport {
             {subtitle}\n\
             -------------------------------\n\
             [{deps_title}]\n\
-            {deps}\n\
+            {cyan}{deps}{end_color}\n\
             -------------------------------\n\
             [{usages_title}]\n\
-            {usages}",
+            {green}{usages}{end_color}",
```

Comment:
nit: The output seems hard to read for me with these colors, maybe we could distinguish the file link from the import with something like yellow (or plain white) for all the `Import: ...` sections?

![image](https://github.com/gauge-sh/tach/assets/10570340/7a786dd0-c53c-47f5-8203-908f23daf12b)